examples/palace-info.py

```diff
@@ -37,6 +37,16 @@
 
     print(f'\nInfo of device {args.device.name!r}:')
     print('ALC version: {}.{}'.format(*args.device.alc_version))
+
+    # TODO: AL info
+    with Context(args.device) as ctx:
+        print('\nAvailable resamplers:')
+        default_idx = ctx.default_resampler_index
+        resamplers = ctx.available_resamplers
+        resamplers[default_idx] += ' (default)'
+        print(*resamplers, sep='\n')
+        print()
```

Comment:
Would be nice if you can remove the surrounding newlines and indent the resamplers like the device names.

Reply:
The newline above too please.  The idea is for `Info of device ...` to look like a continuous block.  Otherwise LGTM.